cmd/genji/dbutil/dump.go

```diff
@@ -64,10 +64,86 @@ func Dump(ctx context.Context, db *genji.DB, w io.Writer, tables ...string) erro
 	return err
 }
 
-// dumpTable displays the content of the given table as SQL statements.
-func dumpTable(tx *genji.Tx, tableName string, w io.Writer) error {
+// dumpTableContent displays the content of the given table as SQL statements.
+func dumpTableContent(tx *genji.Tx, w io.Writer, tableName string) error {
+	// Dump schema first.
+	if err := dumpSchema(tx, w, tableName); err != nil {
+		return err
+	}
+
+	t, err := tx.GetTable(tableName)
```

Comment:
We can avoid getting the table here, since we already have `tableName`. If the table doesn't exist, we'll get an error below anyway